WordPress/Classes/ViewRelated/Reader/Tags View/ReaderTopicCollectionViewCoordinator.swift

```diff
@@ -20,6 +20,10 @@ class ReaderTopicCollectionViewCoordinator: NSObject {
         static let reuseIdentifier = ReaderInterestsCollectionViewCell.classNameWithoutNamespaces()
         static let overflowReuseIdentifier = "OverflowItem"
 
+        static let metrics: ReaderInterestsStyleGuide.Metrics = {
+            return FeatureFlag.readerImprovements.enabled ? .latest : .legacy
+        }()
+
```

Comment:
No issue with this, but I'm going to leave a note since it threw me off when I saw it. This gets initialized only once, so switching the feature flag between `true`/`false` causes the tags to appear incorrect. I had to restart the app for it to be correct.

Reply:
Good point. I don't think I intended to behave this way 😅. At first, it was a computed variable. Then I thought about the amount of creation it can cause from all these accesses (good ol' micro-optimization...), so I decided to turn it into a lazy _instance_ var. Of course I forgot about the "instance" part!

This is now addressed via https://github.com/wordpress-mobile/WordPress-iOS/pull/21736/commits/632910bc33a051130f6ca5b01ba6ead618a88042.